dll/win32/kernel32/winnls/string/format_msg.c

```diff
@@ -99,7 +99,7 @@ static LPWSTR load_message( HMODULE module, UINT id, WORD lang )
 
     if (!module) module = GetModuleHandleW( NULL );
 #ifdef __REACTOS__
-    status = RtlFindMessage(module, (ULONG_PTR)RT_MESSAGETABLE, lang, id, &mre);
+    status = RtlFindMessage(module, (ULONG)RT_MESSAGETABLE, lang, id, &mre);
```

Comment:
`RT_MESSAGETABLE` is defined in `winuser.h` via `MAKEINTRESOURCE`, which makes it a pointer. Casting a pointer to a smaller integer produces a warning on 64 bit, so `ULONG_PTR` seems like the appropriate cast.

Reply:
OK I've just reverted all these ULONG_PTR removals. Build is now fixed.